### PR TITLE
Update release issue steps

### LINKED ISSUE #913: Simplify release process
As @adamralph [suggests](https://github.com/FakeItEasy/FakeItEasy/issues/907#issuecomment-261233206):

> I propose we remove this point from all releases:
>
> - de-list pre-release or superseded buggy NuGet packages if present
>
> The problem here is that it the NuGet client can display erratic behaviour in some cases when a de-listed version is installed in a project. E.g. when uninstalling, it can fail if the package has been de-listed. I suggest we only consider de-listing a package version in exceptional circumstances, e.g. installation causing a major security risk, a malicious version hacked into the gallery by a third party. I.e. stuff that will almost certainly never happen.
>
> - post a link to the GitHub Release in each issue in this milestone, with thanks to contributors
> 
> This feels like overkill for pre-releases. The burden of having to do this for every alpha and beta, plus the noise in notifications, may outweigh the benefit. I wonder if it's better to restrict this to RTM only:
>
> - for an RTM release, post a link to the GitHub Release in each issue in this milestone, with thanks to contributors

Sounds good to me.

### PR DESCRIPTION
**Pull request by blairconrad**
Fixes #913, I think.

1. no need to de-list pre-release packages
2. no need to post link to release in each issue for pre-release versions
3. always attach and publish both nupkgs
4. split out instruction to post tweet here
5. better wording around proper VERSION_SUFFIX environment variable on AppVeyor

### CODE REVIEW

**Comment by blairconrad** [comment]
… before I make the 3.0.0-alpha002 release issue and milestone.

File: rakefile.rb [hunk 1]
```diff
@@ -55,30 +55,48 @@
...
     - create a new issue (like this one) for the next release, adding it to the new milestone
 - [ ] close this milestone
+- [ ] close this issue
+eos
+
+release_issue_prerelease_steps = <<-eos
+Can be labelled **ready** when all other issues destined for this release are closed.
+
+- [ ] run code analysis in VS in *Release* mode and address violations (send a regular PR which must be merged before continuing)
+- [ ] if necessary, change `VERSION_SUFFIX` in [AppVeyor](#{appveyor_server_url}) to "-%version_suffix%" and initiate a build
```
Comment (thomaslevesque) [inline rakefile.rb]: I think "if necessary" isn't necessary here :wink:

**Comment by thomaslevesque** [comment]
Good to squash and merge!

File: rakefile.rb [hunk 2]
```diff
@@ -55,30 +55,48 @@
 repo = 'FakeItEasy/FakeItEasy'
 appveyor_server_url = 'https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/settings'
 release_issue_labels = ['P2', 'build', 'documentation']
-release_issue_common_steps = <<-eos
+release_issue_rtm_steps = <<-eos
```
Comment (blairconrad) [inline rakefile.rb]: now `print_vars` needs updating!

**Comment by blairconrad** [comment]
Don't merge yet!

**Comment by thomaslevesque** [comment]
@blairconrad I approved the latest changes, I'll merge after you squash.

**Comment by blairconrad** [comment]
squashed!

**Comment by thomaslevesque** [comment]
And merged
Thanks @blairconrad!

**Pull request merged**